src/documentation/reference/ReferenceToolkit.tsx

```diff
@@ -74,7 +74,7 @@ const ActiveToolkitLevel = ({
         <List flex="1 1 auto">
           {(module.children ?? []).map((child) => (
             <ListItem key={child.id}>
-              <ReferenceNode docs={child} width="100%" anchor={anchor} mb={0} />
```

Comment:
Matt to check this when reviewing the UI.

I remember needing to add this as reference nodes space themselves out from each other but that shouldn't apply at the top level (only e.g. for fields/methods in a class).

Reply:
We added this back.